### DISCUSSION TITLE
Add "short per second" unit type

### DISCUSSION THREAD

**Discussion by anemyte** [grafana/grafana#53643] [Dashboards, Panels & Visualizations]
Hi there, folks!

I'd like to have a general-purpose unit that represents rate and works in the same fashion as the "short" unit type. In other words, I wish to have a unit that shows on the Y-axis `1M/sec` instead of `1,000,000/sec`.

There are many specialized units, like bytes per second, but no general-purpose one for stuff like rows per second, segments per second, etc. Please add one.

Thanks!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
